glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -152,28 +163,57 @@ open class GleanInternalAPI internal constructor() {
      * @param buildInfo A Glean [BuildInfo] object with build-time metadata. This
      *     object is generated at build time by glean_parser at the import path
      *     ${YOUR_PACKAGE_ROOT}.GleanMetrics.GleanBuildInfo.buildInfo
+     * @param dataPath An optional [String] that specifies where to store
+     *     data locally on the device.
      */
     @Suppress("ReturnCount", "LongMethod", "ComplexMethod")
     @JvmOverloads
     @Synchronized
-    @MainThread
     fun initialize(
         applicationContext: Context,
         uploadEnabled: Boolean,
         configuration: Configuration = Configuration(),
-        buildInfo: BuildInfo
+        buildInfo: BuildInfo,
+        dataPath: String? = null
```

Comment:
I agree, let me give that a try. I originally had it in there and it was causing Android tests to fail for some reason. Then I looked at Python and data_path was included as an initialization parameter, so I changed these two to match.

I guess it makes more sense for Python since it is a requirement and for the mobile SDKs it is more of an option. I will update.